### PR TITLE
Portenta X8: Getting Started update [PC-889]

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- Update getting started tutorial

## What Needs To Be Reviewed
- Content
- Screenshots

## How To Give Feedback
Please leave your feedback as a [Github review](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews). 
You can add comments to specific lines of content / code and ideally use Github's [suggestion feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request). 🙏

### CODE REVIEW

**Review by BenjaminDannegard — COMMENTED** [review]
Left some suggestions. Corrected the Foundries mentions after the examples they gave us on our last article

File: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md [hunk 1]
```diff
@@ -48,27 +47,75 @@ Once its connected you should see the Wi-Fi status dot in the bottom left turn g
...
+
+![Foundries.io Factory dashboard](assets/foundries-factory-dashboard.png)
+
+#### Check Your Device
+
+You can check that your device is fully connected inside your factory by going to the "devices" tab
+
+![Foundries.io factory devices page](assets/foundries-factory-devices.png)
+
+Then chose the device you want to check by clicking on its box and it will open its page.
```
[the remark this answers is not in the document]
Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md]: Fixed on 5531a83

File: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md [hunk 2]
```diff
@@ -48,27 +47,75 @@ Once its connected you should see the Wi-Fi status dot in the bottom left turn g
...
+
+![Register factory button](assets/x8-oob-main-factory.png)
+
+![Factory connection](assets/x8-oob-factory-name.png)
+
+The next panel gives you a code that you need to copy.
+
+![Device factory token](assets/x8-oob-factory-register.png)
+
+Click on the "Complete registration" button  from the Portenta X8 dashboard
```
[the remark this answers is not in the document]
Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md]: Fixed on 5531a83

**Review by BenjaminDannegard — APPROVED** [review]
LGTM

**Pull request merged**